src/images/svg/mastercard.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="84" height="64" viewBox="0 0 84 64">
```

Comment:
please add `aria-labelledby="..."` + `<title>descriptive title<title/>` for all the svgs (see the other svgs for reference)